wiki/Legal/Copyright/en.md

```diff
@@ -0,0 +1,64 @@
+
+**It is osu!'s policy:**
+
+1. to remove or disable access to the content identified in the notice of claimed infringement; 
+
+2. to notify the content provider, member or user that it has removed or disabled access to the content; and 
+
+3. to terminate in appropriate circumstances subscribers and account holders who are repeat infringers.
+
+**Procedure to Supply a Counter-Notice to the Designated Agent:**
+
+If the content provider, member or user believes that the content that was removed or to which access was disabled is not infringing, or the content provider, member or user believes that it has the right to post and use such content from the copyright owner, the copyright owner's agent, or pursuant to the law, the content provider, member or user may send a counter-notice containing the following information to the Designated Agent listed below: 
+
+1. A physical or electronic signature of the content provider, member or user; 
+
+2. Identification of the content that has been removed or to which access has been disabled and the location at which the material appeared before it was removed or access to it was disabled; 
+
+3. A statement under penalty of perjury that the content provider, member or user has a good faith belief that the material was removed or disabled as a result of mistake or a misidentification of the material to be removed or disabled; and 
+
+4. Content provider's, member's or user's name, address, telephone number, and, if available, e-mail address and a statement that such person or entity consents to the jurisdiction of the Federal District Court for the judicial district in which the content provider's, member's or user's address is located, or if the Content provider's, member's or user's address is located outside the United States, for Los Angeles County, California, and that such person or entity will accept service of process from the person who provided notification of the alleged infringement or an agent of such person.
+
+If a counter-notice is received by the Designated Agent, osu! may send a copy of the counter-notice to the original complaining party informing that person that it may replace the removed content or cease disabling it in 10 business days.
+Unless the copyright owner files an action seeking a court order against the content provider, member or user, the removed content may be replaced, or access to it restored, in 10 to 14 business days or more after receipt of the counter-notice, at osu!'s sole discretion.
+
+Please contact osu!'s Designated Agent at the following address: 
+
+Designated Agent to Receive Notification of Claimed Infringement: 
+
+**Email**: [pe@ppy.sh](mailto:pe@ppy.sh) (24 hour response guaranteed) 
+
+\- or -
+
+**Postal Address**: 
+
+<pre>
```

Comment:
Whoops, one of the markdown previewers I used has displayed it with syntax highlighting .-.

I'll a push a change.